pages/deployments/deploying_to_kubernetes.md.erb

```diff
@@ -0,0 +1,192 @@
+# Deploying to Kubernetes
+
+This tutorial teaches deploying to Kubernetes. You'll need to prepare your
+agents with the required packages, cluster access, and Docker registry access
+before hand.
+
+<%= estimated_time "30 minutes" %>
+
+<%= toc %>
+
+## Overview
+
+<%= image "final_test_pipeline.png" %>
```

Comment:
Can we see how this image looks as the last thing in the intro? I think it would be good to have some more words before this one anyhow, even if we just move it a little bit further down.